app/views/users/_bids_table_none.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= t('labels.vendor.account.bids_placed.no_bids_html',
+    index_url: Url.new(link_text: 'current and upcoming auctions', path_name: 'root')) %>
```

Comment:
I might expect to find this link in the view model rather than here (and then we can call the method that contains this logic in the view). Would be easier to I18n-ify the text if we do that, too.